metricbeat/module/docker/diskio/data.go

```diff
@@ -0,0 +1,30 @@
+package diskio
+
+import (
+	"github.com/elastic/beats/libbeat/common"
+)
+
+func eventsMapping(blkioStatsList []BlkioStats) []common.MapStr {
+	myEvents := []common.MapStr{}
+	for _, blkioStats := range blkioStatsList {
+		myEvents = append(myEvents, eventMapping(&blkioStats))
+	}
+	return myEvents
+}
+func eventMapping(myBlkioStats *BlkioStats) common.MapStr {
+	event := common.MapStr{
+		"@timestamp": myBlkioStats.Time,
+		"type":       "blkio",
```

Comment:
What kind of type is that? I not 100% sure if we should use `type` as this seems to be constant across all events. Is it event needed?